pkg/console/operator/operator.go

```diff
@@ -144,23 +143,23 @@ func (c *consoleOperator) Key() (metav1.Object, error) {
 
 func (c *consoleOperator) Sync(obj metav1.Object) error {
 	startTime := time.Now()
-	logrus.Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
-	defer logrus.Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
+	klog.V(4).Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
+	defer klog.V(4).Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
```

Comment:
Shouldn't be neccesary to add newline. Check https://github.com/kubernetes/klog/blob/master/klog.go#L1213-L1235